cmd/activator/main.go

```diff
@@ -158,12 +158,13 @@ func main() {
 		logger.Fatalw("Failed to construct network config", zap.Error(err))
 	}
 
-	// Enable TLS against queue-proxy when internal-encryption is enabled.
-	tlsEnabled := networkConfig.InternalEncryption
-
 	var certCache *certificate.CertCache
 
 	// Enable TLS client when queue-proxy-ca is specified.
+	// right now, all the values of dataplane-trust that aren't disabled do the same thing.
+	// TODO: expand this when the Dataplane trust modes are fully implemented
+	tlsEnabled := networkConfig.DataplaneTrust != netcfg.TrustDisabled
```

Comment:
Do we want to encapsulate this into a function on `networkConfig`?  (Yeah, I realize that requires a PR in `knative/networking`...)